### PR TITLE
fix rhel build

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

File: modules/kie-cloud-operator/install [hunk 1]
```diff
@@ -1,9 +1,5 @@
 #!/bin/sh
 
-INSTALL_PKGS="go-toolset-1.10"
```
Comment (tchughesiv) [inline modules/kie-cloud-operator/install]: This var is used to remove go later in the script.

Comment (spolti) [reply modules/kie-cloud-operator/install]: Sry, missed that.

Yes, the image size is under 400mb:
2019-04-10 09:51:11,496 root         INFO     Original image size: 922.69 MB
2019-04-10 09:51:11,496 root         INFO     Squashed image size: 376.81 MB


Just to be sure, when squashing do you select a layer or squash all layers?

Comment (tchughesiv) [reply modules/kie-cloud-operator/install]: whatever is default in brew

**Comment by tchughesiv** [comment]
@spolti looks like these changes will increase our image size a bit.

currently we're at 394 MB
w/ these changes we bump up to 421 MB

I think its worth the extra 27mb to merge these changes.
/lgtm

**Pull request merged**